agreed/0009-MPID-Messaging/0009-MPID-Messaging.md

```diff
@@ -1,6 +1,6 @@
 - Feature Name: MPID Messaging System
 - Type: New Feature
-- Related Components: [safe_vault](https://github.com/maidsafe/safe_vault), [safe_client](https://github.com/maidsafe/safe_client), [routing](https://github.com/maidsafe/routing)
+- Related Components: [safe_vault](https://github.com/maidsafe/safe_vault), [safe_client](https://github.com/maidsafe/safe_client), [mpid_messaging](https://github.com/maidsafe/mpid_messaging)
 - Start Date: 22-09-2015
```

Comment:
The repository has not been created yet, the link should work when that's done. Yes, a separate crate for messaging.